sagemaker/run/realtime_entry_points/node_prediction_entry.py

```diff
@@ -0,0 +1,429 @@
+
+def transform_fn(model,
+                 request_body,
+                 request_content_type,
+                 response_content_type='application/json'):
+    """ An end-to-end function to handle one request from input to output.
+
+    #TODO: add the API specification url here:
+
+    According to GraphStorm real-time inference API specification, the payload is like:
+
+    {
+        "version": "gs-realtime-v0.1",
+        "gml_task": "node_classification",
+        "graph": {
+            "nodes": [
+                {
+                    "node_type": "author",
+                    "features": {
+                        "feat": [
+                            0.011269339360296726,
+                            ......
+                        ]
+                    },
+                    "node_id": "a4444"
+                },
+                {
+                    "node_type": "author",
+                    "features": {
+                        "feat": [
+                            -0.0032965524587780237,
+                            .....
+                        ]
+                    },
+                    "node_id": "s39"
+                }
+            ],
+            "edges": [
+                {
+                    "edge_type": [
+                        "author",
+                        "writing",
+                        "paper"
+                    ],
+                    "features": {},
+                    "src_node_id": "p4463",
+                    "dest_node_id": "p4463"
+                },
+                ......
+            ]
+        },
+        "targets": [
+            {
+                "node_type": "paper",
+                "node_id": "p4463"
+            },
+            or
+            {
+                "edge_type": [
+                        "paper",
+                        "citing",
+                        "paper"
+                    ]
+                "src_node_id": "p3551",
+                "dest_node_id": "p3551"
+            }
+        ]
+    }
+
+    Parameters
+    ----------
+    model: a tuple of three elements
+        The output of model_fn, including a model object, a graph construct config dict, and a
+        GSConfig object.
+    request_body: str
+        The request payload as a JSON string. The JSON string contains the subgraph for inference.
+    request_content_type: str
+        A string to indicate what is the format of the payload. For GraphStorm built-in real-time
+        input function, the format should be 'application/json'.
+    response_content_type: str
+        A string to indicate what is the format of the response. Default is 'application/json'.
+
+    Returns
+    -------
+    res: JSON format of GSRealTimeInferenceResponseMessage
+        The JSON representations of an instance of GSRealTimeInferenceResponseMessage.
+    response_content_type: str
+        A string to indicate what is the format of the response.
```

Comment:
Mention we only support `application/json` currently?

Reply:
yes. add it to the doc string.